closure/goog/testing/net/xhrio.js

```diff
@@ -453,6 +469,13 @@ goog.testing.net.XhrIo.prototype.send = function(
   if (this.testQueue_) {
     this.testQueue_.enqueue(['s', url, opt_method, opt_content, opt_headers]);
   }
+
+  // not interested in what factory does to create object. Contention is simply
```

Comment:
[Complete sentences should use appropriate capitalization.](https://google.github.io/styleguide/javascriptguide.xml?showone=Comments#Comments)